### DISCUSSION TITLE
Service단과 Controller단의 구분

### DISCUSSION THREAD

**Discussion by june0216** [EFUB-LakkuLakku/LakkuLakku-Back#105] [General]
저는 예전에 Controller와 Service 레이어의 구분을 명확하게 인지하지 못한 상태로 코드를 작성했었습니다. 그래서 컨트롤러에서 데이터를 읽어오는 부분도 있는 것 같은데  "_데이터 저장 계층으로부터 읽어온 데이터를 사용하고 변경하는 등의 작업을 하는 곳이 서비스 계층_"이라는 점을 생각하여 서비스 계층과 컨트롤러 계층의 구분을 명확하게 하려고 코드를 수정하고 있습니다! (예를 들어, 컨트롤러에서 데이터를 읽어오는 부분을 서비스 계층으로 옮겨서 작성)

**Comment by nitronium102** [comment]
+ service단에서 불러오면 @transactional로 한 번 검증할 수 있기 때문에 카카오모빌리티, 쿠팡, 닥터나우 등에서는 service 단을 통해서 검증한다고 합니다!
